### PR TITLE
Update index.adoc

### PR DESCRIPTION
**Pull request by zackrw**
Add link to [OpenZeppelin Contracts Security Center]

### CODE REVIEW

**Comment by Amxx** [comment]
IMO we should also add that to the readme.

**Review by Amxx — DISMISSED** [review]

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -45,6 +45,8 @@ To keep your system secure, you should **always** use the installed code as-is,
 
 Please report any security issues you find via our https://www.immunefi.com/bounty/openzeppelin[bug bounty program on Immunefi] or directly to security@openzeppelin.org.
 
+The https://contracts.openzeppelin.com/security[OpenZeppelin Contracts Security Center] contains more details about the secure development process.
```
Comment (ernestognw) [inline docs/modules/ROOT/pages/index.adoc:48]: I checked in the deployment preview and the name sounds a bit repetitive in the site, how do we feel about replacing by the following:

```suggestion
The https://contracts.openzeppelin.com/security[Security Center] contains more details about the secure development process.
```

Also, have we consulted with marketing if we should use the full "Open Zeppelin"?

**Review by ernestognw — APPROVED** [review]
Don't want to block for this

**Review by ernestognw — APPROVED** [review]
LGTM, thanks @zackrw!

**Pull request merged**